portal-ui/src/screens/Console/Buckets/ViewBucket/ViewBucket.tsx

```diff
@@ -867,6 +886,15 @@ const ViewBucket = ({
                 idField="name"
               />
             </TabPanel>
+            <TabPanel index={3} value={curTab}>
+              <TableWrapper
+                columns={[{ label: "User", elementKey: "accessKey" }]}
+                isLoading={loadingEvents}
```

Comment:
This loading needs to be related to the loading used to fill this table, in this case loading users
```suggestion
                isLoading={loadingUsers}
```